make/scripts/update_copyright_year.sh

```diff
@@ -49,66 +51,74 @@ rm -f -r ${tmp}
 mkdir -p ${tmp}
 total=0
 
-# Default or supplied company name
-if [ "$3" != "" ] ; then
-  company="$3"
-fi
-
-# This year or supplied year
-if [ "$2" != "" ] ; then
-  year="$2"
-else
-  year=`date +%Y`
-fi
-
-# VCS select
-vcs="$1"
+usage="Usage: `basename "$0"` [-c company] [-y year] [-h|f]"
+Help()
+{
+  # Display Help
+  echo "Updates the Copyright year range in Git sources."
+  echo
+  echo "By default, the tool limits the processed changesets "
+  echo "to those in the current branch and the current year."
+  echo
+  echo "Note, cancelling the script will skip cleanup in /tmp."
+  echo
+  echo $usage
+  echo "options:"
+  echo "c     Specifies the company. Set to Oracle by default."
+  echo "y     Specifies the copyright year. Set to current year by default."
+  echo "h     Print this help."
+  echo "f     Updates all change sets in a full year."
```

Comment:
- Please specify the leading dash on all options
- I would print "y" after "f" and write something along these lines "If -f is specified, defines the year of changes for which to modify the copyright (current year if omitted)"

Reply:
Hi Thomas, 

I assume with this we hope to clarify what time period of changes we are addressing with `-f`. Would something like this communicate what you suggested? 

```
Usage: update_copyright_year.sh [-c company] [-y year] [-h|f]
options:
-c     Specifies the company. Set to Oracle by default.
-y     Specifies the copyright year. Set to current year by default.
-f     Updates the copyright for all change sets in a given year,
       as specified by -y.
-h     Print this help.
```